### ISSUE TITLE
Windows 10: NewRandomAccessFile failed to Create/Open: Access is denied

### ISSUE THREAD

**Issue by Jaketack** [ildoonet/tf-openpose#24]
Hello everyone,

I tried to execute the inference.py and realtime_webcam.py on Windows 10 in Anaconda Prompt 5.0.1
First question, does this version work on Windows? Maybe that's the problem already.

I manually downloaded the training data from the download.sh of the pretrained folder and inserted it into a new folder \models\trained\mobilenet_368x368\model-release

The CMD gives me this error:

> 2017-11-28 14:09:44.802667: W C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\framework\op_kernel.cc:1192] Data loss: Unable to open table file .\models\trained\mobilenet_368x368\model-release: Unknown: NewRandomAccessFile failed to Create/Open: .\models\trained\mobilenet_368x368\model-release : Access is denied.
; Input/output error
2017-11-28 14:27:33.631622: W C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\util\tensor_slice_reader.cc:95] Could not open .\models\trained\mobilenet_368x368\model-release: Unknown: NewRandomAccessFile failed to Create/Open: .\models\trained\mobilenet_368x368\model-release : Access is denied.
> ; Input/output error

This error message comes several times and then some other errors appears too, but I think they are constrained to the posted error.

Anyone has an idea, why this happens? I tried to open Anaconda Prompt with Administrator rights too.

Thank you!

EDIT: It works with cmu but not with mobilenet. So maybe is the file missing? I only downloaded mobilenet_v1_0.50_224.ckpt.*** with some other extension, not mobilenet_v1_0.50_224.**ckpt**

**Comment by callemein** [comment]
Hello @Jaketack, I'm facing the same issue on linux and opened a issue:

https://github.com/ildoonet/tf-openpose/issues/23

**Comment by Jaketack** [comment]
@callemein thank you, I think it is a bit different. But I will close this issue, so it's clearer.

**Comment by lucheng07082221** [comment]
@Jaketack i have the same problem,can you tell me the solution, thank you!

**Comment by iggy12345** [comment]
so, did anybody find a solution to this?
